xfields/ibs/_equilibrium_emittance.py

```diff
@@ -241,83 +294,70 @@ def compute_emittance_evolution(
         )
         # If emittance_coupling_factor is non zero, then natural emittance is
         # modified accordingly
+        # fmt: off
         if emittance_coupling_factor != 0 and emittance_constraint.lower() == "coupling":
-            # The convention used is valid for arbitrary damping partition
-            # numbers and emittance_coupling_factor.
-            emittance_y = (
-                emittance_x
-                * emittance_coupling_factor
-                / (
-                    1
-                    + emittance_coupling_factor
-                    * twiss.partition_numbers[1]
-                    / twiss.partition_numbers[0]
-                )
-            )
-            emittance_x *= 1 / (
-                1
-                + emittance_coupling_factor
-                * twiss.partition_numbers[1]
-                / twiss.partition_numbers[0]
-            )
+            # The convention used is valid for arbitrary damping partition numbers and emittance_coupling_factor.
+            emittance_y = emittance_x * emittance_coupling_factor / (1 + emittance_coupling_factor * twiss.partition_numbers[1] / twiss.partition_numbers[0])
+            emittance_x *= 1 / (1 + emittance_coupling_factor * twiss.partition_numbers[1] / twiss.partition_numbers[0])
 
         if emittance_coupling_factor != 0 and emittance_constraint.lower() == "excitation":
             # The convention used only enforce a constraint on the vertical
             # emittance
             emittance_y = emittance_x * emittance_coupling_factor
     else:
         emittance_x, emittance_y, emittance_z = initial_emittances
-
+        # fmt: on
+    # ----------------------------------------------------------------------------------------------
+    # Handle initial longitudinal emittance and potential effect of bunch lengthening
     sigma_zeta = (emittance_z * twiss.bets0) ** 0.5
     sigma_delta = (emittance_z / twiss.bets0) ** 0.5
-    if input_sigma_zeta is not None:
-        warnings.warn(
-            "'input_sigma_zeta' is specified, make sure it remains "
-            "consistent with 'initial_emittances'."
-        )
-        sigma_zeta = input_sigma_zeta
-    elif input_sigma_delta is not None:
-        warnings.warn(
-            "'input_sigma_delta' is specified, make sure it remains "
-            "consistent with 'initial_emittances'."
-        )
-        sigma_delta = input_sigma_delta
+    if overwrite_sigma_zeta is not None:
+        warnings.warn("'overwrite_sigma_zeta' is specified, make sure it remains consistent with 'initial_emittances'.")
+        sigma_zeta = overwrite_sigma_zeta
+    elif overwrite_sigma_delta is not None:
+        warnings.warn("'sigma_delta' is specified, make sure it remains consistent with 'initial_emittances'.")
+        sigma_delta = overwrite_sigma_delta
```

Comment:
This is just renaming of the `input_sigma_[delta,zeta]` and warning on one line, nothing else.